spring-cloud-huawei-sample/swagger-demo/edge-service/src/main/java/com/huaweicloud/sample/EdgeMain.java

```diff
@@ -0,0 +1,15 @@
+package com.huaweicloud.sample;
```

Comment:
Do we need to add LICENSE to all files ?

Reply:
done